src/main/java/com/google/sps/servlets/MetadataStore.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.util.List;
+
+public interface MetadataStore {
+  abstract void storeData(Metadata data);
```

Comment:
You don't need the "abstract" keyword here. Also all the methods here should be public.

Reply:
I ran into an issue with this earlier actually. Whenever I put just public and not abstract, it ended up giving me an error saying that I had to explicitly write abstract, otherwise it would not compile.

Reply:
Hmmm. We never use abstract in Google. What is the error message?
FYI:
https://stackoverflow.com/questions/641536/why-would-one-declare-a-java-interface-method-as-abstract

Reply:
Hmmm.. for whatever reason, I'm no longer getting that error. When I changed these methods to not include "abstract", I didn't get the same issue anymore.